### PR TITLE
New drink - virgin mojito

### PR DESCRIPTION
**Pull request by FarhiaM**

### CODE REVIEW

File: src/recipes/virgin-mojito.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        "Crush and mix the mint leaves and lime juice in a glass.",
+        "Add the raw sugar into the mix.",
+        "Mix it together with the honey syrup.",
+        "Add sparkling water.",
+        "Garnish with some slices of lime and a spring of mint."
+    ],
+    "image": "virgin-mojito.jpg",
+    "keywords": [
+       "Virgin mojito",
+       "Non-alcoholic drinks",
```
Comment (alfg) [inline src/recipes/virgin-mojito.json]: Can the keywords be just `virgin` and `non-alcoholic`?

Comment (FarhiaM) [reply src/recipes/virgin-mojito.json]: Yes, sure! I'll fix it

**Comment by FarhiaM** [comment]
I have now updated the PR with the correct keywords and image size. Thank you for the feedback :)

File: src/recipes/virgin-mojito.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        "Crush and mix the mint leaves and lime juice in a glass.",
+        "Add the raw sugar into the mix.",
+        "Mix it together with the honey syrup.",
+        "Add sparkling water.",
+        "Garnish with some slices of lime and a spring of mint."
+    ],
+    "image": "virgin-mojito.jpg",
+    "keywords": [
+       "virgin",
+       "non-alcoholic",
```
Comment (alfg) [inline src/recipes/virgin-mojito.json]: Hi @FarhiaM, sorry one more thing if you can remove this comma so it doesn't break json.
```suggestion
       "non-alcoholic"
```

Comment (FarhiaM) [reply src/recipes/virgin-mojito.json]: Yes, I'll remove the extra comma

**Comment by FarhiaM** [comment]
I've updated the PR now and removed the extra comma 🙂

**Comment by alfg** [comment]
Great! Thanks for all the fixes and the PR! 💯

**Pull request merged**